backend/Squirrel.Core/Squirrel.Core.Common/DTO/Users/UpdateUserNamesDTO.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Squirrel.Core.Common.DTO.Users;
+
+public class UpdateUserNamesDTO
```

Comment:
we already named all our DTOs with 'Dto' postfix, so stick to this naming (instead of 'DTO'). Also rename files

Reply:
fixed